src/plugin/places/places.js

```diff
@@ -110,12 +110,13 @@ export const SourceFields = [
   Fields.LAT_DMS,
   Fields.LON_DMS,
   Fields.MGRS,
+  Fields.ALT,
   Fields.SEMI_MAJOR,
   Fields.SEMI_MINOR,
   Fields.SEMI_MAJOR_UNITS,
   Fields.SEMI_MINOR_UNITS,
-  Fields.TIME,
-  Fields.ORIENTATION
+  Fields.ORIENTATION,
+  RecordField.TIME
```

Comment:
Was using the wrong time field before causing some issue?

Reply:
The `TIME` column doesn't exist on Saved Places until app refresh, because the KML exporter saves it here:
https://github.com/ngageoint/opensphere/blob/0cbf7c7a2e00f8e20913abbdbd2cc13055c179b2/src/os/ui/file/kml/abstractkmlexporter.js#L600-L602

If you create a new Place and open Feature Info for it, the `TIME` value will be blank because that column doesn't exist. Feature Info will display the `recordTime` field as `TIME` so using that column here fixes the Feature Info display issue.

It looks like the Show Features grid does not do that mapping however. I wonder if we should run a time mapping to add the `TIME` column instead of this change.

Reply:
I updated the column to use `RecordField.TIME` as the `field` and `Fields.TIME` as the `name`. This is consistent with how the time column is typically treated on vector sources (see `os.source#addDefaults`), and with the field `featureedit.js` is actually setting for the time.